### ISSUE TITLE
Additional info: Make legs expandable [FE]

### ISSUE THREAD

**Issue by MRTNKCR** [kiwicom/smart-faq#513]
Based on this issue: https://github.com/kiwicom/smart-faq/issues/487

The expandable functionality has been merged, now we need to add the additional info.

Acceptance criteria:

arrows on right
additional info is shown upon expanding the leg
![41983536-af266dd4-7a2e-11e8-8a5a-2b62d01a20c6](https://user-images.githubusercontent.com/7661448/42225160-2785e8ca-7edc-11e8-8b32-b4e0de75c9c4.png)

When expanded:
![41857742-8267dc8e-7898-11e8-8cae-c4e349c43257](https://user-images.githubusercontent.com/7661448/42225174-2ea263c2-7edc-11e8-863e-fe4cb70fbe33.png)

Storybook on FE: https://bit.ly/2Kjxmcy

**Comment by kantord** [comment]
The arrows on the right seem to be working on @vborszeki's implementation already, if I understand correctly what you mean by them https://github.com/kiwicom/smart-faq/pull/509

**Comment by vborszeki** [comment]
That's right! This issue is just about adding the remaining info to the expanded list, which is not available now from the GraphQL server. @kantord

**Comment by MRTNKCR** [comment]
@vborszeki @kantord should I create one more issue for the backend part of this in the GraphQL repo?

**Comment by vborszeki** [comment]
@MRTNKCR  yes, it would be nice. 🙂

**Comment by MagdalenaAlmanova** [comment]
🥇
